e2e-tests/development-runtime/plugins/gatsby-plugin-top-layout/top-layout.js

```diff
@@ -5,7 +5,15 @@ import CssBaseline from "@material-ui/core/CssBaseline";
 import { ThemeProvider } from "@material-ui/styles";
 import theme from "../../src/theme";
 
+let injectFirstNode;
+
 export default function TopLayout(props) {
+  if (typeof window !== `undefined` && !injectFirstNode) {
```

Comment:
@oliviertassinari I would not expect that here. What do you think about:
```javascript
// gatsby-browser.js
export const onClientEntry = () => {
  const head = document.head;
  const injectFirstNode = document.createComment(`mui-inject-first`);
  head.insertBefore(injectFirstNode, head.firstChild);
};
```

Reply:
Agree, not the ideal place for this logic. Ok let me try `onClientEntry` :).